### ISSUE TITLE
amenity=research_institute

### ISSUE THREAD

**Issue by danstowell** [gravitystorm/openstreetmap-carto#2077]
The [amenity=research_institute](http://wiki.openstreetmap.org/wiki/Tag:amenity=research_institute) tag is fairly new (122 uses at time of writing), but seems uncontroversial and has been used all over the world ([overpass link](http://overpass-turbo.eu/?w=%22amenity%22%3D%22research_institute%22+global&R)).

Today I "fixed" a research institute that had been tagged as amenity=university ([changeset](http://www.openstreetmap.org/changeset/37648541)). Of course this kinda removed its status in the osm-carto rendering.

Research institutes are fairly similar to universities, so may I suggest that they could be rendered in the same way? This would be useful to incentivise mappers to tag them less ambiguously. I know the tag is not yet massive, but I'd like to see what you think. Cheers.

**Comment by pnorman** [comment]
122 uses is insufficient to be an established tag.

**Comment by matkoniecz** [comment]
Especially as it is a new tag - tagging scheme may improve.

**Comment by geozeisig** [comment]
Please render [amenity=research](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dresearch_institute) institute in the same way as university. The wiki page has been redesigned, so it's clearer now how the tag should be applied. There are only 350 applications, but if `research_institute` has no color, `landuse=commercial` is often added. That does not belong together.

**Comment by kocio-pl** [comment]
How is it different from much more popular https://wiki.openstreetmap.org/wiki/Tag:office%3Dresearch (which would be rendered soon by #3061)?

**Comment by geozeisig** [comment]
`office=research` is a building or a part of a building (node). `amenity=research_institute` is a campus (area with several institute buildings). It would be good if both are considered. [Example](https://www.openstreetmap.org/way/216369067)

**Comment by kocio-pl** [comment]
Interesting case - I'm not sure about the meaning for buildings/campuses, but at least this is consistently documented. Still the usage is too low (far less than ~2k) and while we have no latest data, growing it is a slow process (~100/year):

![taghistory 18](https://user-images.githubusercontent.com/5439713/38171473-b000eed8-359a-11e8-889b-2712a071139d.png)

**Comment by dieterdreist** [comment]
sent from a phone

> On 1. Apr 2018, at 10:30, kocio-pl <notifications@github.com> wrote:
> 
> How is it different from much more popular https://wiki.openstreetmap.org/wiki/Tag:office%3Dresearch (which would be rendered soon by #3061)


+1, office is for a office, amenity is for the institution (the space it occupies/where it is located). You could also have 2 institutions in the same building, or they could have different offices in the same facility.

**Comment by dieterdreist** [comment]
sent from a phone

> On 1. Apr 2018, at 10:54, kocio-pl <notifications@github.com> wrote:
> 
> Interesting case - I'm not sure about the meaning for buildings/campuses, but at least this is consistently documented. Still the usage is too low (far less than ~2k)


300 research institutions are not few, especially as they are often significant features. Do you know how many research institutions there are in the world?

**Comment by kocio-pl** [comment]
I don't know, but it's ~130 in Poland:

https://forumakademickie.pl/informator-fa/instytuty-badawcze/

**Comment by geozeisig** [comment]
In Germany, there are 25,000. Not all will be relevant. See [Research Explorer](http://www.research-explorer.de/research_explorer.en.html?).
I will work to get it more.

**Comment by kocio-pl** [comment]
Great!

I guess some of them can be just too small to be tagged as  `amenity=research_institute` and instead may be tagged as `office=research`, due to our scheme definitions, but still 2k limit doesn't sound unreasonable. 

But thanks for a question, @dieterdreist. There can be some important objects with lower numbers in reality and we should treat such cases in a special way.

**Comment by dieterdreist** [comment]
sent from a phone

> On 2. Apr 2018, at 02:29, kocio-pl <notifications@github.com> wrote:
> 
> I don't know, but it's ~130 in Poland:
> 


currently there 349 in OSM, this is +25% since the “end” of taginfo history = a few months

**Comment by kocio-pl** [comment]
Nice, that's encouraging. I still hope for regular data updates, but even manual quarterly updates would be good: https://github.com/tyrasd/taghistory/issues/10#issuecomment-377773123.

The question is how would we like to render such objects?

**Comment by geozeisig** [comment]
The color like university would be good. Something else is good too, maybe a bit darker?

**Comment by Tomasz-W** [comment]
Yellow amenity=university fill + label would be ok.
Another option is to discuss about some blue fill + dark-blue label for office areas and research institutes.

**Comment by geozeisig** [comment]
Fixes #2077
Add rendering for `office=research`.
The color like amenity=university fill + label would be good.

**Comment by geozeisig** [comment]
Can we open a PR? I don't know what to do.

**Comment by kocio-pl** [comment]
I feel it's still too early, first we need some numbers to be sure.

**Comment by polarbearing** [comment]
I agree that university-yellow would be reasonable for a campus. 
If it is just a building outline or just a node, how would the label look like?

**Comment by geozeisig** [comment]
`amenity=university`  has no icon if it was mapped as a node or building. So I would say university-yellow and the name is enough. 
If it's just a point or building you should use `office=research`. Which icon we use for that we should discuss in a new issue.

**Comment by matkoniecz** [comment]
> amenity=university has no icon if it was mapped as a node or building.

In that case it is relatively unimportant and name should be sufficient.

**Comment by geozeisig** [comment]
The name is the most important. The color is also needed otherwise `landuse=commercial` is used.

**Comment by polarbearing** [comment]
636 uses today

**Comment by jeisenbe** [comment]
Note that the previous proposal for the similar tag amenity=research_institution was rejected back in 2009 and 2010: https://wiki.openstreetmap.org/wiki/Proposed_features/research_institution - rejection of research_institution=yes
https://wiki.openstreetmap.org/w/index.php?title=Proposed_features/research_institution&oldid=281785 - rejection of amenity=research_institution

It may be useful to read through those old objections to the tag, though most are just opposed to the key `amenity`. 

Perhaps it is still not clear how office=research and amenity=research_institute should be distinguished.

**Comment by polarbearing** [comment]
You find my comments on the talk pages of the proposals. The only voting was for amenity=research_institution, the proposal box was later changed to research_institution=yes which did not have a voting. 

Anyway, this issue is not about the old attempts, it is about the **amenity=research_institute** which is growing organically since 2015.

To answer your question:
* An office is e.g. in a building where I can sit at a desk.
* A campus of a research institute can potentially hold significant technical installations besides typical office buildings. Examples are a Particle Accelerator, a Research reactor, or a test drive area for research on autonomous driving. The latter are not offices.

**Comment by jidanni** [comment]
Maybe https://wiki.openstreetmap.org/wiki/Tag:amenity=research%20institute should warn that this feature is not currently rendered at this time. At least that is what happened with
https://en.wikipedia.org/wiki/Wuhan_Institute_of_Virology .

**Comment by polarbearing** [comment]
A feature being rendered or not in a particular style is not a hazard someone has to be "warned" about. Go and map more institutes and the chances to render it increase. 

The institute is in the database, so Nominatim [is able to find it](https://www.openstreetmap.org/search?query=Wuhan+Institute+of+Virology).

**Comment by jidanni** [comment]
It finds a blank area. Maybe I should add a comment to the changeset asking the user to try some different tags? This place is all in the news. Senators are talking about it. Shouldn't there be a way for it to be on the map?

**Comment by polarbearing** [comment]
No you should not recommend somebody to tag something incorrect to get a feature rendered.

**Comment by jidanni** [comment]
I see. I should
>  Go and map more institutes and the chances to render it increase.

**Comment by ognone** [comment]
I would like to suggest landuse=research, which is also not uncommon in the database.

**Comment by jeisenbe** [comment]
If you wish to request rendering of landuse=research that will need to be a [new issue](https://github.com/gravitystorm/openstreetmap-carto/issues/new) - but that tag has only been used 97 times and was not discussed, and the wiki page was just created today. 

The rejection of a proposal for amenity=research_institution in 2008 does not have much influence, since it was for a different tag and it was a long time ago.

**Comment by jeisenbe** [comment]
Original reason for closing was low usage (122 times), now this has increased to 746 occurences in the database, so I've reopened for discussion.

**Comment by jidanni** [comment]
> I would like to suggest [landuse=research](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dresearch), which is also not uncommon in the database. And completely unrejected if this matters to you.

Thanks! This is a lifesaver. I was almost about to ask the  HOTSM team. Worldwide crisis POI like this cannot wait. Death tolls are rising.

**Comment by HolgerJeromin** [comment]
> Thanks! This is a lifesaver. [...] Worldwide crisis POI like this cannot wait. Death tolls are rising.

You are right! 
Having this research institute rendered on an map millions of Chinese people are using every day will save thousands of lives!

(I really hope ill people will go to a hospital and not to a [BSL-4 facility](https://en.wikipedia.org/wiki/Biosafety_level#Biosafety_level_4) itself.)

**Comment by pnorman** [comment]
I don't see that 746 occurrences is enough to warrant rendering. What are people thinking would be a suitable rendering?

**Comment by HolgerJeromin** [comment]
The most logical choice would be the same as rendering an university.

**Comment by polarbearing** [comment]
Yes that has been suggested by several people in this thread above. Which would be in sync with other societal amenities (#1624) and an easy fix.

**Comment by imagico** [comment]
I see a distinct lack of consistent use of this tag and a poor definition so i would suggest to close this again.

Specifically

* the definition is completely tied to the meaning of the fairly vague English language term *research*.
* the geometric semantics are very unclear.  Sometimes the tag is used on buildings, sometimes on (often non-verifiable) wrapper polygons around buildings or groups of buildings, sometimes there are research institutes tagged within research institutes.
* consistency in use is very poor.  In Munich for example there are two feature with the tag - one of them (https://www.openstreetmap.org/way/103038933) is a place where no actual research is performed, it is an administrative and training center - while there are certainly dozens of research facilities within the city that are tagged differently.

From a purely cartographic perspective if a tag is used equally for research offices and things like farmland patches used for agricultural research rendering that in a meaningful way without other information is not really possible.

**Comment by polarbearing** [comment]
What is vague about the term "research"? 

OxDict/BrE: "the systematic investigation into and study of materials and sources in order to establish facts and reach new conclusions."
Or, on the practical side, "[Horizon 2020](https://ec.europa.eu/programmes/horizon2020/what-horizon-2020) is the biggest EU Research and Innovation programme ever with nearly €80 billion of funding available over 7 years (2014 to 2020)"

**Comment by polarbearing** [comment]
> the geometric semantics are very unclear. Sometimes the tag is used on buildings, sometimes on (often non-verifiable) wrapper polygons around buildings or groups of buildings, sometimes there are research institutes tagged within research institutes.

Yes this is real life. It is not different from schools or universities. Sometimes they are just a single building without outdoor area, sometimes a separate institute/department is a tenant within a larger institution, sometimes the facility has a campus and sometimes not.

**Comment by DnaX** [comment]
I will add some examples of the usage of amenity=research_institute in italy:

- CNR-Pisa: [National Research Council](https://en.wikipedia.org/wiki/National_Research_Council_(Italy)): https://www.openstreetmap.org/way/23082352

- [Laboratori Nazionali del Gran Sasso](https://en.wikipedia.org/wiki/Laboratori_Nazionali_del_Gran_Sasso), one of the most important physics laboratories of the world: https://www.openstreetmap.org/way/79401619

- Bruno Kessler Fondundation: https://www.openstreetmap.org/way/813142998

They are closed areas with buildings, parking lots and various structures, such as schools or universities. But it can also be a single point where the structure is not recognizable as an area, like an "office" (but it is not an office) of a large building with other functions.

Ps. now the global usage it's > 1k items.

**Comment by velikodsky** [comment]
Global usage is already growing faster. This year the number of uses has doubled (now ~ 1.5k). I suggest rendering like universities.

**Comment by polarbearing** [comment]
As I see none of the previous concern being valid anymore, please go ahead and render as discussed.

**Comment by jeisenbe** [comment]
Unfortunately the previous concerns appear to be valid still.

The features with this tag can be an office in a building, a building in a university or hospital, a campus of many office buildings, a campus with office buildings and industrial buildings, or an area of farmland which is used for “research”, in it’s many meanings. 

I checked https://overpass-turbo.eu/s/130J for England, the Netherlands, Belgium, Switzerland, and New Zealand.

Although there are some examples like the ones shown above, there is not one consistent type of meaning for this feature: some are within an amenity=university and are a building or several buildings or just an office in a University building with other functions.

There are only a half-dozen in all of Southwest England, including London, Oxford and Cambridge, surprisingly. Often they are just small offices like the “Overseas Development Institute”: https://www.openstreetmap.org/node/7205812400 https://www.odi.org - or https://www.openstreetmap.org/node/8287236858 the "Thrombosis Research Institute” in the National Heart and Lung building: https://www.tri-london.ac.uk/

There is an Institute of Zoology building with this tag in the London Zoo: https://www.openstreetmap.org/way/40234936

This building in the Wellcome Genome Centre is the "Wellcome Trust Sanger Institute” - https://www.openstreetmap.org/relation/9398629 - but the whole campus is landuse=commercial (https://www.wellcomegenomecampus.org/)

Near Oxford, the "Diamond Light Source Synchrotron” is tagged - https://www.openstreetmap.org/relation/548974 - but the surrounding area is landuse=industrial with the name Rutherford Appleton Laboratory, and the larger area is the Harwell campus - it seems to contain non-profit research institutes as well as commercial / industrial businesses: https://www.harwellcampus.com/organisation-list/

The Pera Business Park appears to be an office building but it’s tagged amenity=research_institute: https://www.openstreetmap.org/way/221951953 - https://www.perabusinesspark.co.uk

Robert Aitken Institute for Clinical Research is within a hospital campus, sharing a building with the Haematology department: https://www.openstreetmap.org/node/7207998548 

Kroto Research Institute is tagged both amenity=research_institute and office=university and it’s within the Univeristy of Sheffield: https://www.openstreetmap.org/way/201385725

Others are clearly industrial areas with large works (factory) buildings which happen to be used for developing new industrial processes. E.g. https://www.openstreetmap.org/node/8346073097 "Fair-Fusion” inside of a Emmtec Emmen industrial chemical area -https://chemicalparks.eu/companies/emmtec

And there are examples where the area is farmland which happens to be used for agricultural investigations: https://www.openstreetmap.org/way/197943816 - related to https://www.openstreetmap.org/way/874675602#map=18/50.98056/3.77901 (Flanders Research Institute for Agriculture, Fisheries and Food) - https://ilvo.vlaanderen.be/en

Many of these research institutes are partners with Universities and some actual have classrooms: 
https://www.openstreetmap.org/way/798023590 - https://www.field-studies-council.org/locations/eppingforest/ - “The purpose built, fully accessible Centre has six classrooms, and associated facilities, with direct access into our teaching sites which include woodland, grassland, heath and river. Most of our teaching sites are fully accessible."

This is a couple of buildings in the middle of a proteced moor (heathland) accessible via a track: https://www.openstreetmap.org/way/836619083 "Moor House”

“Dairy Campus”, a research dairy producing milk products in the Netherlands - https://www.openstreetmap.org/way/678477984 - https://www.dairycampus.nl/en/Home.htm "At Dairy Campus innovative projects and activities are carried out in order to generate new information and knowledge to drive innovation in the dairy chain.”

So as @Imagico had noted before, this tag is used for all kinds of places which might be called an “institute”, whether they are parts of a University or part of an Industrial facility, or separate. And it is used for many features which are focused on research, whether it is basic scientific research, or agricultural development, or industrial processes improvement, or commercial research, etc.

**Comment by 1ec5** [comment]
Indeed, these research facilities are in quite a range of settings. However, at a glance, it doesn’t seem that much more diverse than `amenity=college` or `amenity=university` facilities, some of which are just a few rooms in a building and some of which constitute an entire town. One of the larger ones I’ve mapped is [a radio observatory](https://www.openstreetmap.org/relation/12122500). It even has an airstrip on site. Counterintuitively, much of the site does render, due to a protected area (protecting against radiointerference), but that zone is known by a different name and isn’t as well-known.

**Comment by dieterdreist** [comment]
sent from a phone

> On 31 Jan 2021, at 23:39, Joseph E <notifications@github.com> wrote:
> 
> The features with this tag can be an office in a building, a building in a university or hospital, a campus of many office buildings, a campus with office buildings and industrial buildings, or an area of farmland which is used for “research”, in it’s many meanings.


an area of farmland isn’t an “institute” though, nor are buildings (but they can house institutes, and with ambiguous shortcut tagging it could be tagged together with a building tag set, so they might be “ok”)

It’s ok if there are different scales of the same thing (an office in a building vs. a campus with several buildings)

**Comment by polarbearing** [comment]
Most of the alleged inconsistencies are generic mapping mistakes and could be cited for lots of objects. 

Imagine the following scenario: Assume amenity=school is not rendered and proposed to do so, and now we get inconsistencies reported that sometimes a campus is mapped, sometimes a building and sometimes just a node in a building for a driving school. Would that stop us from rendering schools?

Some reseach facilities tagging might be misplaced, or some have other landuse tags to force them being rendered - we have this style as feedback to the mappers, including to reveal mistakes.

**Comment by polarbearing** [comment]
Research and education are interwoven, there is no sharp boundary in reality, as @jeisenbe' analysis has shown. Thus objects might carry both taggings, research facilities might be placed on university campuses, research facilities might have some seminar rooms. All these mixtures. Yes. 
This is the reason the _identical_ rendering was proposed. Anyway there are research institutes that are not universities, some of them geographically quite large, and they are denied rendering. Strange.

**Comment by AgusQui** [comment]
It already has more than 2000 uses https://taginfo.openstreetmap.org/tags/amenity=research_institute#overview, How many do you need to decide to render?

**Comment by velikodsky** [comment]
It should be rendered.
![taghistory (1)](https://user-images.githubusercontent.com/24278321/151431697-ef2c90e8-c5ff-4f80-8ea2-bea7f91abf86.png)

**Comment by vachan-maker** [comment]
Hi
Research Institute is still not rendered
Example: https://www.openstreetmap.org/way/1033208665

**Comment by matkoniecz** [comment]
>  Research Institute is still not rendered

Yes, which is already indicated by fact that this issue is open.

**Comment by vachan-maker** [comment]
haha😁
I thought this issue is declined because of the decline tag on this issue

**Comment by matkoniecz** [comment]
Oh, good point! It was not removed on reopening it

**Comment by bgo-eiu** [comment]
I support this!

**Comment by BryannaM** [comment]
Use continues to grow it is well above 3,000 and it drives me nuts facilities like NBAF don't render. NBAF is a very significant facility. There might not be a huge number in total of these types of facilities because the ones there are are significant. 

![image](https://user-images.githubusercontent.com/22663346/211160050-439c67d1-89d7-478b-8a0c-cf2c3fbf8759.png)
![image](https://user-images.githubusercontent.com/22663346/211160113-5baf0771-ea59-4c7a-b26c-a57a76ebfcb6.png)

**Comment by Caseyb87** [comment]
We are now at >3,600 uses of this tag and it has an iD editor preset. The wiki page documents this tag as de facto (though that was a fairly recent edit). It seems to me this is distinct enough from ``office=research`` (e.g., you wouldn't tag [NASA GSFC](https://www.openstreetmap.org/relation/4237285) as an office).

I support rendering. Suggest rendering in the same fashion as a university.

**Comment by maro-21** [comment]
4599 uses (2024-05-05)

This is one of the few commonly used tags for which I miss the rendering. Since such institutes often cover large areas, people purposely use other tags just to make them appear on the map. Frequently used substitutes are `amenity=university` or `landuse=commercial`. Many research instututes in my city don't have this tag, precisely for this reason - because they have `amenity=university`.
Is there anything preventing you from adding this rendering?

**Comment by imagico** [comment]
> Is there anything preventing you from adding this rendering?

Views of OSM-Carto maintainers have been explained in:

https://github.com/gravitystorm/openstreetmap-carto/issues/2077#issuecomment-587386908
https://github.com/gravitystorm/openstreetmap-carto/issues/2077#issuecomment-770464009

If anything of substance has changed in mapping practice since then please present that here.

The key `research=*` has been introduced and is used 773 times now but it only provides thematic differentiation, meaning it provides information on the topic of research, not the type of research activity practiced (i.e. it does not allow distinguishing between farmland used for agricultural research, a research lab doing analytics and a bunch of offices).

`office=research` is used 12.5k times - no clear semantic delineation seems to exist in either practical use or documentation between that and `amenity=research_institute`.

**Comment by scaidermern** [comment]
> `office=research` is used 12.5k times - no clear semantic delineation seems to exist in either practical use or documentation between that and `amenity=research_institute`.

Usually research institutes - at least the ones that I know - consist of many different offices and often contain related facilities (laboratories, sometimes smaller lecture halls, common areas like dining rooms used by different research groups together etc). There is no other known tag to me that would fit here.

**Comment by novolife** [comment]
I'm shocked that this repo guys can refuse to such an important feature because of some insignificant "problems", and even consider that they are protecting justice and we're the evil guy who want to destroy the "beautiful" carto. Honestly I won't give a shit on this style if it's not the default rendering of osm.

**Comment by dreua** [comment]
@novolife I've been thinking about a fork recently, there is a handful of issues very similar to this where I feel very similar to what you said. We should probably find a different space to discuss this further and not hijack this issue here.

**Comment by TonyJC99** [comment]
I tagged multiple campuses as being government-owned research institutes and was shocked to see that they've yet to be given the yellow "education" render on the OSM map. This is now an 8 year old issue apparently! Yes, these campuses exist and hold multiple buildings in them with differing purposes. No, they're not universities nor owned by them. In the case of my use for them, these are owned by government ministries, and I'm sure that there are a variety of them that can vary from the public and private sector. 

I do believe that this tag, as a landuse and amenity, is justified in being rendered, not because of its nature, but because its use is still increasing year by year. It now has 4.5k uses, and still shows robust growth, despite the fact that some people apparently wish to undervalue its use despite the fact that it's not buildings we're talking about here, but whole plots of land!

![OSM](https://github.com/gravitystorm/openstreetmap-carto/assets/51808669/f3a670d7-d2ed-4712-826d-1ef3759a00ba)

**Comment by dch0ph** [comment]
> > `office=research` is used 12.5k times - no clear semantic delineation seems to exist in either practical use or documentation between that and `amenity=research_institute`.
> 
> Usually research institutes - at least the ones that I know - consist of many different offices and often contain related facilities (laboratories, sometimes smaller lecture halls, common areas like dining rooms used by different research groups together etc). There is no other known tag to me that would fit here.

I agree with the argument here. There are lot of research institutes of all shapes and sizes, but the merit of using the same area (but not building) rendering as education-focussed organisations is that it focusses on the larger better-defined institutes.

I think the `office=research` tag is much more confused. It seems to be used for individual laboratories or departments within something bigger:

![image](https://github.com/gravitystorm/openstreetmap-carto/assets/20069699/bc486b40-c2c8-4145-b3da-aae233530405)
[Here the Kroto Research Institute has been tagged as `office=university`, probably to get a name rendered. ]

Hence it is not surprising that there are lots of `office=research`, but frankly these are much less interesting / important than research institutes.

I have reviewed the comments made by Carto maintainers above, but I don't find the arguments convincing against the specific proposal of rendering in the same way as educational institution, not least because it is irrelevant whether a particular entity does research and/or teaching.

The objection that farmland can be part of a research institute is also unconvincing. If you have an enclosed land parcel that is part of a "campus", then it is part of the institute. If it is some random field _operated_ by a research institute, then it is not.

**Comment by michaelblyons** [comment]
Now above 5000 uses. Meanwhile, `office=research` has gone from 12k to 13k.

Meanwhile, I continue to see dual-tagging as `landuse=commercial` just to put a name on the (sometimes rather large) institutions.

**Comment by imagico** [comment]
Has now 6.1k uses, about 800 of which are also tagged as building.  For comparison: `office=research` now has 14.5k uses.

A renewed analysis of tag use world wide might be useful, but it would have to critically look at the practical meaning, in particular what actual land uses the tag is used for.  Previous analysis and the wiki indicate the tag is used for indicating land that is operated by some organization performing some type of research (in some sense of the term *research*) - mixing a large scope of actual land uses as we otherwise render them in OSM-Carto.  That makes it difficult to develop a suitable rendering that does not provide wrong mapping incentives.  It is clear that most polygons tagged `amenity=research_institute` qualify for another tag that we already render - depending on the actual land use - like `landuse=commercial` (for research offices and laboratories), `landuse=farmland` (for agricultural research) etc.  This does not rule out rendering `amenity=research_institute` in general, but it rules out rendering it in a way that implies a specific land use different from those.  That makes it different from the educational landuses.  It is clear consensus among mappers that an agricultural research farmland is not suitably tagged with `amenity=university`, even when operated by a university.  That does not seem the case for `amenity=research_institute`.

**Comment by RyanWatern** [comment]
> amenity=research_institute

Personally, I have used `amenity=research_institute` for forensic institutes, as they do not fit cleanly under any other existing tag that I am aware of. The same applies to cancer research centers that focus exclusively on research and do not provide patient treatment; as such, they are neither clinics nor hospitals, making `amenity=research_institute` an appropriate and accurate classification.

Additionally, I have mapped marine research institutions that are dedicated solely to the scientific study of marine life, as well as facilities focused on solar energy, space research, and other forms of medical research, (Brain, Stem Cell, Etc). In many of these cases, I have found it difficult to identify a more suitable or precise tagging alternative without feeling that I made an error or that it was misleading.

As said in another forum, this tag has now been used 6,089 times, with 3,835 instances (63%) mapped as areas, and around 85% of all `amenity=research_institute` tags being added within the past five years. Given this massive increase, I feel many in the community would appreciate a worldwide analysis to better describe what this tag would be used for and the designation of a rendering.

**Comment by 1ec5** [comment]
For what it’s worth, id-tagging-schema has long assigned an Erlenmeyer flask to both `office=research` and `amenity=research_institute`. The idea is to evoke scientific research broadly speaking using a very recognizable shape, without necessarily implying that the facility specializes in chemistry or biology.
